### DISCUSSION TITLE
Roses are red, this category is for those that are NEW … 🌹 [New to GitHub Check-in]

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#46889] [New to GitHub]
Consider this check-in a valentine 💌 from me to all of you~ 

I love reading all your discussions and seeing y’all help each other! Continue to spread the love this Valentine’s Day by answering other members’ questions, leaving a heart on product feedback you want to see, or thanking a member who has been rocking it in the community.  

🌟**Community Superstars**: A round of applause for these community citizens 👏 
* @surjendu104: Provided multiple workflows that were easy to understand, and additional links for more information when answering a question about [problems pushing to a local repository from remote GitHub](https://github.com/orgs/community/discussions/45698). Nicely done! 
* @shayan1234554321: For a detailed answer that went above and beyond answering a user’s [question](https://github.com/orgs/community/discussions/46502) on repository permissions. 
* @chris-c-thomas: Their response on [maintaining project files between two computers](https://github.com/orgs/community/discussions/46018) was a delight to read, it was personable and easy to follow. 

😈 **Can You Hack It?** Get involved with the open source community by joining an upcoming Hackathon. 
* TransportHacks: Join your fellow hackers on February 17th to create hacks that help resolve issues with the shipping and supply chain industry 🚢. There are both [North America and EU](https://organize.mlh.io/participants/events/9007-transporthacks) and [Asia/Pacific](https://organize.mlh.io/participants/events/9007-transporthacks) friendly events! 
* ELC Accessible Beauty Hackathon: Ideate and develop accessible and inclusive solutions for the challenges that people with disabilities face in their beauty experience 💄 from February 4 - March 20th. [Learn more](https://elchackathon.devpost.com/?ref_feature=challenge&ref_medium=discover). 
* Global Hack Week: Spend March 6-13 hacking with the Major League Hacking community at GHW Social Good Week 😀! [Get the details](https://organize.mlh.io/participants/events/9151-global-hack-week-social-good-week). 
* ​​What the Hack – DevOps with GitHub (In person): Join us for a set of challenge-based hackathons in London on March 14. Attendees work in small teams to solve a series of technical challenges. [Register here](https://msevents.microsoft.com/event?id=197908456&ocid=AID3052689). 

🆒 **Cool stuff™️** happening around GitHub and the Community
* **How the GitHub Docs team uses GitHub Projects**: [Explore](https://github.blog/2023-02-02-how-the-github-docs-team-uses-github-projects/) how the GitHub Docs ✏️ team uses GitHub Projects for content coordination, reviews, and publishing.
* **Category Forms on GitHub Discussions**: Maintainers of GitHub repositories can now use Category Forms to create templates ☑️ for their Discussions - find out [more](https://github.blog/changelog/2023-02-03-category-forms-on-github-discussions/).
* **Ludum Dare 52 staff picks** 🕹: We’ve got the ten top games from the latest Ludum Dare game jam plus source code for you to [check out](https://github.blog/2023-02-03-ludum-dare-52-staff-picks/). Pun intended.

Planning on attending any Hackathons mentioned above or elsewhere? Let us know in the comments!

**Comment by jhonathan-humnel** [comment]
Great post, feeling the happnies.
